src/NUnitEngine/nunit.engine.core/nunit.engine.core.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <RootNamespace>NUnit.Engine</RootNamespace>
-    <TargetFrameworks>net20;netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net20;netstandard2.0;netcoreapp3.1;net5.0</TargetFrameworks>
+    <NoWarn>$(NoWarn);SYSLIB0011;SYSLIB0012</NoWarn><!-- TODO: Get rid of obsolete stuff -->
```

Comment:
How much is obsolete? Should we use local pragmas to disable warnings so we can find them or are there too many?

Reply:
I started to do pragmas but it kept popping up in new places. Once we get rid of the net20 build, I think we can get rid of this. That implies (1) making the console runner use some flavor of .NET 4 and (2) using a pluggable agent to run under .NET 2.0. I see both of those as still being a few months down the road though.